src/utils/argTypes.ts

```diff
@@ -0,0 +1,193 @@
+export const footerArgTypes = {
+  footerTitle: {
+    name: 'title',
```

Comment:
This is not used in cinnamon, it is only for storybook, therefore it should not be in cinnamon utils folder.

Reply:
the file is now in utils on stories folder.